src/ui/MessageContent/index.scss

```diff
@@ -181,9 +177,9 @@
   .sendbird-message-content__left {
     box-sizing: border-box;
     display: inline-flex;
-    justify-content: flex-end;
+    align-items: flex-start !important;
     position: relative;
-    width: 50px;
+    width: 52px !important;
```

Comment:
Is there a need for the !important I figured, it wouldn't be required as this is actually in the library.